docs/releases/1.9.txt

```diff
@@ -1280,3 +1280,5 @@ removed in Django 1.9 (please see the :ref:`deprecation timeline
 * Private API ``django.test.utils.TestTemplateLoader`` is removed.
 
 * The ``django.contrib.contenttypes.generic`` module is removed.
+
+* The `enable_comments` property `FlatPage` is removed from the `FlatPageAdmin`.
```

Comment:
This note goes under "Backwards incompatible changes" (the section are you in now is features that went through a deprecation cycle.
Please limit line length in docs to 79 chars.
Please use double-backticks (single doesn't have any effect on the rendered content).
property -> field
"FlatPage.enable_comments field is..."